GetStream/stream-chat-swift
# [CIS-2295] Add support for hiding connection status with `isInvisible`

### 🔗 Issue Links

https://github.com/GetStream/stream-chat-swift/issues/2368
CIS-2295

### 🎯 Goal

The goal of this task is to be able to hide the connection status when connecting a user and also expose this property to the current user.

### 🛠 Implementation
- Adds `UserInfo.isInvisible` to be able to set the value when connecting the user
- Adds `CurrentUserDTO.isInvisible` to be able to get the value when the user is already connected

### 🧪 Manual Testing Notes
- Open a channel with User A
- With another device, before connecting User B, tap the Configuration Button in Demo App
- Enable `isInvisible` in the user info
- Open a channel with the User B
- User A should no see User B has online in the Channel Header View

### ☑️ Contributor Checklist
